R/cr_cn.r

```diff
@@ -37,6 +37,14 @@
 #'   `http://data.medra.org`. DOI agency is checked first (see
 #'   [cr_agency()]).
 #'
+#' @section caching:
+#' 
+#' Caching is done using the package \pkg{webmiddens}. To cache requests
+#' first enable webmiddens `webmiddens::wm_enable()`, then configure where to
```

Comment:
why can't it be in one step i.e. why couldn't `wm_enable()` have an argument with configuration?

Reply:
and why is it `webmiddens::wm_enable()` and not `rcrossref::cr_wm_enable()`?

Reply:
In webmiddens docs in practice I don't really see what enabling means. :-)

Reply:
> why can't it be in one step i.e. why couldn't wm_enable() have an argument with configuration?

because I thought users may want to change the cache path separately from enabling caching (in addition to other settings such as expiration time), but do you think that wouldn't be a the case?

Reply:
> and why is it webmiddens::wm_enable() and not rcrossref::cr_wm_enable()?

because in my vision of how webmiddens will be used, minimal changes by package developers will make webmiddens easier to use, and for users it will be easier to just know that they have to enable webmiddens with one function across packages rather than each package using webmiddens having their own enable/config fxns

Reply:
> I don't really see what enabling means.

i'll explain that better

Reply:
:+1: